### PR TITLE
Announcement release 1.4.4

### PR DESCRIPTION
**Pull request by renatocaval**

### CODE REVIEW

File: src/blog/lagom-1-4-2-and-1-3-11.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Lagom team has released the versions 1.4.2 and 1.3.11. For a list of the spe
 
 This release provides several bug fixes and improvements. As an overview, this release contains the following:
 
-- Fixes a bug in loading settings from the `LagomClientFactory`. Special thanks to [two10](/two10) for the detailed report and bugfix suggestion. ([#1256](https://github.com/lagom/lagom/pull/1256))
+- Fixes a bug in loading settings from the `LagomClientFactory`. Special thanks to [two10](https://github.com/two10) for the detailed report and bugfix suggestion. ([#1256](https://github.com/lagom/lagom/pull/1256))
```
Comment (ignasi35) [inline src/blog/lagom-1-4-2-and-1-3-11.md]: 👏

File: src/blog/lagom-1.4.3.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+This release provides a fix for [#1250](https://github.com/lagom/lagom/issues/1250). 
+
+[#1250](https://github.com/lagom/lagom/issues/1250) only affects Lagom projects making usage of JDBC persistence layer, being it for the journal and/or for read-side tables. 
+
+This release upgrades the versions of the underlying Slick library to `3.2.3` and the `akka-persistence-jdbc` plugin to version `3.3.0`.
+
+## Slick 3.2.3 special note
+
+Since Slick 3.2.2, the `maxConnections` must equal `numThreads`. This is to prevent some situations where a deadlock could occur. A warning will be emitted in case `maxConnections > numThreads` and a runtime exception will be thrown if `maxConnections < numThreads`. Therefore the best possible configuration is `maxConnections == numThreads`. See https://github.com/slick/slick/pull/1855 for detailed information.
```
Comment (ignasi35) [inline src/blog/lagom-1.4.3.md]: is `https://github.com/slick/slick/pull/1855` converted to a link automagically? If not, can we make it a link please?

File: src/blog/lagom-1.4.3.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+If you haven't changed the settings for your `async-executor`, you have nothing to do. However, if you have configured your `aync-executor` with other values, you should review it and make sure that `maxThreads == maxConnections`.
+
+See GitHub for the [full list of commits](https://github.com/lagom/lagom/compare/1.4.2...1.4.3) and the issues and pull requests assigned to the [1.4.3 milestone](https://github.com/lagom/lagom/milestone/27?closed=1).
+
+
+## Updating a Lagom project to version 1.4.3
+
+To update an sbt project, change the version of `lagom-sbt-plugin` in your `project/plugins.sbt` file.
+
+To update a Maven project, change the `lagom.version` property in your top-level `pom.xml`.
```
Comment (ignasi35) [inline src/blog/lagom-1.4.3.md]: 👍

File: src/blog/lagom-1.4.3.md (hunk 2 above)
Comment (renatocaval) [reply src/blog/lagom-1.4.3.md]: Done.

**Comment by ignasi35** [comment]
Feel free to merge when artifacts are ready.

**Comment by renatocaval** [comment]
Made a mistake when release 1.4.3 and we need now to skip to 1.4.4

**Pull request merged**